src/docs/welcome/quickStart/DOC.md

```diff
@@ -89,31 +94,31 @@ Bootstrapped application with essential tools for rapid development
 1. Run the container with default options
 
     ```sh
-    bash scripts/run.sh
+    SPANDX_CONFIG=path/to/insights-frontend-starter-app/config/spandx.config.js sh path/to/insights-proxy/scripts/run.sh
     ```
 
-#### More options for running
+##### More options for running
 
 * Run with the local Legacy Insighs API
```

Comment:
"local Legacy Insighs API" we dont want to document this anymore I think right?
Also Insights is misspelled

Reply:
Removing